### ISSUE TITLE
Support Ticket mail auto creation

### ISSUE THREAD

**Issue by ericzen** [SythilTech/Odoo9#14]
Hello,

Is-it possible to create a support ticket on the reception of an email ?

I've created a user helpdesk with an email and created a reception rules for this account with Actions to Perform on Incoming Mails : Create a New Record Website Support Ticket.

But nothing happened.

Thanks for your help 

Ericzen

**Comment by SythilTech** [comment]
Hi Ericzen,


Thank for for submitting this feature idea.

I have released a new version, you should now be able to auto create new 
support tickets from an email using the actions to perform on incoming mail.

Regards
Steven

**Comment by ericzen** [comment]
Sorry but i don't understand how i can do to create a new support ticket from email.

**Comment by ericzen** [comment]
Ok i've find the solution. That's great thank you. 🥇 

A other question : Is it possible to have a default mail reponse for the mail send. 

Indeed, if I decide to create a user "helpdesk" and want that mailbox was the only one to receipt helpdesk mail, but can answer with an other user, i want that the response send by my customer arrived in helpdesk mailbox.

Thanks for your help.

**Comment by SythilTech** [comment]
Hi,


You can modify the from / reply-to in the "Suppport Ticket Reply 
Wrapper" email template if you want all responses to go to helpdesk email.


Regards

Steven

**Comment by ericzen** [comment]
Ok i've changed the value as indicated.
But know a last question : I've created a default value on ticket creation for Category. And associated my user (user@domain) to this category. When i received a mail on helpdesk@domain, the creation is ok with the default category. But my user, user@domain doesn't receive a mail from odoo to inform that the ticket is under the category.
How can i have a automatic mail to user@domain when the ticket is created with this category or less without category ?

**Comment by ericzen** [comment]
PS : You're work is really wondefull and full usable.

**Comment by SythilTech** [comment]
Hi,


Sorry I can't reproduce this issue.

Please note the email template responsible for the category email is 
"Suppport Ticket New Ticket (Category)", it is coded to send the email 
from your company email to the users partner email of all users assigned 
to the category.

Please check under settings->technical->email for the reason it failed 
to deliver the email.


Regards

Steven

**Comment by SythilTech** [comment]
Hi,


Thank you, it is always good to see people finding my modules helpful.


Regards

Steven

**Comment by ericzen** [comment]
Hello.
There's some problems with emails. 
First : When i send a mail to helpdesk@domain.td, the ticket is not created. But when i see my imap box, the mail is considered as read. In odoo nothing appears in the list of message in Settings/technical/mail.
Second : When i create a ticket on "Support tickets", impossible to input a description or a subjet. So the ticket have "false" as name. So impossible to create a task.
Third : Even the same things, if i change category no mail is send to the member of this category.

**Comment by ericzen** [comment]
Other demand : how can i parameter a sequence number for ticket ?

**Comment by ericzen** [comment]
So this is what i think. If you have ever received a mail from a contact, the other mail send by this contact to create a new issue is reading by odoo but not imported.
I've test to send a new helpdesk by a new mailbox and the first mail is imported. The other not. ???

**Comment by ericzen** [comment]
Hello !

I've made new tests and this is the results : 

1 - When you create a support on website, you can input description and subject. This is not the case if you create it directely in odoo in Sales/Tickets Support/ Customer support.
When you create this support on website, the user who create the demand, received a confirmation mail. If you change the category, users associated to the category received an email.

2 - If you receive a mail to open automaticaly a new support, the customer don't received a confirmation mail. No message in Settings/Technical/mail. So no action by odoo.
If you defined a default parameter for Category, this value is associated to the support ticket when you received the mail but no mail was transfered to users associated to the category. No message in Settings/Technical/mail.

3 - ever the same question about sequence number for ticket.

Thanks for your help

**Comment by ericzen** [comment]
Hello,

An other issue : 
I've changed in mail models "Suppport Ticket New" and "Suppport Ticket New Ticket (Category)', the From value to associate "helpdesk@domain.tld". I've controled the change is ok.
I create a new support ticket on the website and submit my demand. When i return in mail models, the value is automaticaly replace by "contact@domain.tld". 

Why ? Contact is for leads and helpdesk for support.

Regards

**Comment by SythilTech** [comment]
Hi,Thank you for reporting this issue, I will be extremely busy with my jobs this week but can get back to you on the weekend.RegardsStevenOn 17 Jan 2017 7:50 PM, ericzen <notifications@github.com> wrote:Hello,
An other issue :
I've changed in mail models "Suppport Ticket New" and "Suppport Ticket New Ticket (Category)', the From value to associate "helpdesk@domain.tld". I've controled the change is ok.
I create a new support ticket on the website and submit my demand. When i return in mail models, the value is automaticaly replace by "contact@domain.tld".
Why ? Contact is for leads and helpdesk for support.
Regards

**Comment by ericzen** [comment]
Sorry,

I don't understand your answer. all questions are what i've done in odoo, not on Github. 

i close this tread and create one with my questions.

**Comment by SythilTech** [comment]
Hi,


First: Yes this is a bit of a quirk of the incoming mail in v9, it only 
creates records under settings/technical/mail if the mail isn't read 
first from say an external mail client, this ultimately prevents the 
ticket from being created.

Second: Sorry this was meant to be disabled since when a ticket is 
created via the website form it sends out the new ticket in your 
category email and a few other small things.

Third: See second point.


Regards

Steven

**Comment by SythilTech** [comment]
Hi,


Sorry not sure what you mean by parameter a sequence number for a ticket.

I have released a new version which let's you  change the next ticket 
number under the edit company form view.


Regards

Steven

**Comment by SythilTech** [comment]
Hi,


yea sorry the form controller kinda took over that, It uses the company 
email by default, that is where "contact@domain.tld 
<mailto:contact@domain.tld>" was coming from.


I've released a new version which should allow you to modify the from 
address in those templates without it being overwritten by the form 
controller.


Regards

Steven
